Cue/app/actions/library.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+import type { PromiseAction } from './types';
+import LibraryApi from '../api/Library';
+
+async function loadLibrary(state): PromiseAction {
+  let library = eval(await LibraryApi.fetchLibrary(state));
```

Comment:
Why do we need this `eval`? Could we not just do `response => response.json()` in line 19 of `LibraryApi`?

I'm concerned that using `eval` on the result of a network call could be a security risk.